build.gradle

```diff
@@ -322,8 +325,8 @@ tasks.register("generateBstGrammarSource", JavaExec) {
     description = 'Generates BstLexer.java and BstParser.java from the Bst.g grammar file using antlr3.'
 
     inputs.dir('src/main/antlr3/org/jabref/bst/')
-    outputs.dir("src/main/generated/org/jabref/logic/bst/")
-    args = ["-o", "$projectDir/src/main/generated/org/jabref/logic/bst/" , "$projectDir/src/main/antlr3/org/jabref/bst/Bst.g" ]
+    outputs.dir("src-gen/main/java/org/jabref/logic/bst/")
+    args = ["-o", "src-gen/main/java/org/jabref/logic/bst/" , "$projectDir/src/main/antlr3/org/jabref/bst/Bst.g" ]
```

Comment:
```suggestion
    args = ["-o", "$projectDir/src-gen/main/java/org/jabref/logic/bst/" , "$projectDir/src/main/antlr3/org/jabref/bst/Bst.g" ]
```

Reply:
It is not added to `outputs.dir`. To avoid cluttering the `build.gradle` file, I would leave this change out and "just" use the relative path. (And it works with the relative path).

I'll keep it in my head though.